### PR TITLE
Add black

### PR DESCRIPTION
**Pull request by jrcastro2**

### CODE REVIEW

File: invenio_pages/views.py [hunk 1]
```diff
@@ -74,34 +70,34 @@ def render_page(path):
...
+        return extra["wrapped"](exception)
     else:
         return exception
 
 
 def _add_url_rule(url_or_urls):
     """Register URL rule to application URL map."""
-    if isinstance(url_or_urls, six.string_types):
+    if isinstance(url_or_urls, str):
         url_or_urls = [url_or_urls]
```
Comment (Pineirin) [inline invenio_pages/views.py:99]: Why this change?

**Review by Pineirin — APPROVED** [review]
:+1:

File: invenio_pages/views.py (hunk 1 above)
Comment (jrcastro2) [reply invenio_pages/views.py:99]: Six is a Python 2 and 3 compatibility library, but we only support python 3 thus is not needed.

File: setup.cfg [hunk 2]
```diff
@@ -57,6 +57,7 @@ install_requires =
 [options.extras_require]
 tests =
     pytest-cache>=1.0
+    pytest-black>=0.3.0,<0.3.10
```
Comment (ntarocco) [inline setup.cfg:60]:
```suggestion
    pytest-black>=0.3.0
```

File: .git-blame-ignore-revs [hunk 3]
```diff
@@ -0,0 +1 @@
+dc9c80d4d1142d5de37c133c15261fd1d006dcba
```
Comment (ntarocco) [inline .git-blame-ignore-revs:1]: if you push to fix the comment below, you will have to change this commit id.
When merging, the maintainer/merger should make sure that the commit id is not changed.

**Review by ntarocco — APPROVED** [review]
